tox.ini

```diff
@@ -56,8 +56,8 @@ commands =
 # that exit status. Then run it again on the highest severity and confidence
 # levels and get our pass/fail status from that.
 # This is reported to bandit at https://github.com/PyCQA/bandit/issues/341
-    - bandit -s B404,B410 -i -l -r playbooks/
-    bandit -s B404,B410 -iii -lll -r playbooks/
+    - bandit -s B404,B410,B501 -i -l -r playbooks/
```

Comment:
It would be good to capture why we're skipping these codes.

Reply:
added comments regarding skip of B501